src/Symfony/Component/Cache/Adapter/PhpArrayAdapter.php

```diff
@@ -0,0 +1,379 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Cache\Adapter;
+
+use Psr\Cache\CacheItemInterface;
+use Psr\Cache\CacheItemPoolInterface;
+use Symfony\Component\Cache\CacheItem;
+use Symfony\Component\Cache\Exception\InvalidArgumentException;
+
+/**
+ * Adapter building static PHP files that will be cached by OPCache.
+ * This Adapter is in read-only if you use AdapterInterface methods.
+ * You can use the method "warmUp" to build the cache file.
```

Comment:
I suggest:
```
 * Caches items at warm up time using a static PHP array that is stored in shared memory by OPCache since PHP 5.6.
 * Warmed up items are read-only and run-time discovered items are cached using a fallback adapter.
```